src/storable.rs

```diff
@@ -594,6 +708,54 @@ fn encode_size(dst: &mut [u8], n: usize, bounds: &Bounds) {
     }
 }
 
+fn decode_size_universal<T>(src: &[u8]) -> usize
```

Comment:
(communication) Again, related to method names. The term "universal" is not clear, and raises more questions than answers. Consider renaming this method to just `decode_size` and rename the current `decode_size` to `decode_size_of_bound`.